vue-app-example1/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>E-commerce Example Application</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+</head>
+
+<body>
+    <div id="app">
+        <div class="container">
+            <br>
+            <nav id="top-navigation" class="well well-sm flex flex-row align-center">
+                <a href="#" @click="isShowingCart = false"><strong>E-commerce Inc.</strong></a>
+
+                <div class="text-right pull-right cart-info">
+                    <span class="stats">{{ cart.items.length }} <template v-if="cart.items.length == 1">item</template> <template v-else>items</template> in cart, totalling {{ cartTotal | currency }}</span>
+                    <button class="btn btn-primary" @click="isShowingCart = true">View Cart</button>
+                </div>
+            </nav>
+
+            <div v-if="!isShowingCart" id="products" class="row list-group">
+                <div v-for="product in products" class="item col-xs-4">
```

Comment:
cand folosesti v-for, ii bine sa folosesti un key="product.id". Daca vue nu are un key, atunci nu stie de care element sa se lege Vue cand trebuie sa faca modificari asupra acelui element.